### PR TITLE
optimize consensus code,modify return type when no error msg

### PR DESCRIPTION
**Pull request by xiemylogos**

### CODE REVIEW

File: consensus/vbft/event_timer.go [hunk 1]
```diff
@@ -190,7 +187,7 @@ func (self *EventTimer) getEventTimeout(evtType TimerEventType) time.Duration {
 //
 // internal helper, should call with lock held
 //
-func (self *EventTimer) startEventTimer(evtType TimerEventType, blockNum uint32) error {
+func (self *EventTimer) startEventTimer(evtType TimerEventType, blockNum uint32) {
```
Comment (Honglei-Cong) [inline consensus/vbft/event_timer.go]: suggest changing the panic to 'return err'

**Review by Honglei-Cong — COMMENTED** [review]
overall, looks good to me.
except one case: err should be returned on 'start timer'

File: consensus/vbft/event_timer.go (hunk 1 above)
Comment (xiemylogos) [reply consensus/vbft/event_timer.go]: ok

**Review by Honglei-Cong — APPROVED** [review]
looks good.

**Pull request merged**